b92.html

```diff
@@ -1,12 +1,36 @@
 
 <html>
-   <head>
-     <title>B92 - Internet, Radio i TV stanica - najnovije vesti iz Srbije i sveta</title>
-     <link rel="stylesheet" type="text/css" href="style.css" />
-   </head>
-   <body>
-     <div id="top">
-      <h2>B92</h2>
-     </div>
-   </body>
+<head>
+	<title>B92 - Internet, Radio i TV stanica - najnovije vesti iz Srbije i sveta</title>
+	<link rel="stylesheet" type="text/css" href="style.css" />
+</head>
+<body>
+	<div id="top">
+		<h2>B92</h2>
+		<ul>
+			<li><a href="http://www.b92.net/novo/index.php">NOVO</a></li>
+			<li><a href="http://www.b92.net/info/">INFO</a></li>
+			<li><a href="http://www.b92.net/sport/">SPORT</a></li>
+			<li><a href="http://www.b92.net/biz/">BIZ</a></li>
+			<li><a href="http://superzena.b92.net/">SUPERZENA</a></li>
+			<li><a href="http://www.b92.net/zivot/">ZIVOT</a></li>
+			<li><a href="http://www.b92.net/kultura/">KULTURA</a></li>
+			<li><a href="http://www.b92.net/automobili/">AUTO</a></li>
+			<li><a href="http://www.b92.net/tehnopolis/">TEHNOPOLIS</a></li>
+			<li><a href="http://www.b92.net/putovanja/">PUTOVANJA</a></li>
+			<li><a href="http://www.b92.net/zdravlje/">ZDRAVLJE</a></li>
+			<li><a href="http://www.b92.net/ljubimac/">LJUBIMAC</a></li>
+			<li><a href="http://www.b92.net/esports/">ESPORTS</a></li>
+			<li><a href="http://www.b92.net/">VISE</a></li>
+		</ul>
+	</div>
+	<div class="nba">        
+		 <h6 class="yellowheader"><a href="http://www.b92.net/info/">SPORT</a></h6>
```

Comment:
You still have problems with indentation, spaces and tabs are mixed. Mark this row from beginning to `<h6` and you'll see.